index.scss

```diff
@@ -1,13 +1,37 @@
 .micro-header {
+  position: relative;
   padding: 9px;
   text-align: center;
   background-color: hsl(134, 100%, 29%);
 }
 
+.micro-header--white {
+  padding: 9px;
+  text-align: center;
+  background-color: #fff;
+}
+
 .micro-header__logo {
   display: inline-block;
   width: 42px;
   height: 22px;
   text-indent: -99999px;
   background: url('./img/white.svg') no-repeat;
+
+  .micro-header--white & {
+    // White bg has green logo
+    background: url('./img/green.svg') no-repeat;
+  }
+}
+
+.micro-header__lhs {
+  position: absolute;
```

Comment:
No but I can try. to see how it works out. The thing is it needs to float all the way to the edge of the screen, won't the grid constrain it to the container width?